src/shared/bg_parse.cpp

```diff
@@ -607,6 +607,7 @@ void BG_ParseBuildableAttributeFile( const char *filename, buildableAttributes_t
 
 	text = text_buffer;
 
+	ba->dretchAttackable = false; //FIXME
```

Comment:
> Fix what?

I agree, please tell what is to be fixed, why it's perfectible.

Reply:
Fixed: basically this is already the default, because the struct is initilised to 0 (I don't remember if that's with an explicit memset or with a `= {};`, but it doesn't matter)